pkg/engine/engine.go

```diff
@@ -278,7 +278,7 @@ func (eng *Eng) DeleteAllObjects(ctx context.Context) error {
 		for _, name := range objInfo.Names {
 			err := eng.dynamicClient.Resource(objInfo.GVR).Namespace(ns).Delete(ctx, name, deletions)
 			if err != nil {
-				return err
+				eng.log.Info("Warning: cannot delete object", "name", name, "err", err.Error())
```

Comment:
Adding the error info. is helpful, but should it return `err` if it's not nil?

Reply:
Not entirely. We still need to check for timeout. Updated the code.